content/topics/python-specific/python-backend-dev-environment-setup/_index.md

```diff
@@ -0,0 +1,48 @@
+An IDE normally consists of at least a source code editor, build automation tools, and a debugger. What makes an IDE so useful is the I: integrated. You could use just about anything for a development environment and many people use a variety of basic, individual programs in place of an IDE but an integrated environment gives you the ability to do everything in a single editor.  
+- [python3.7](https://linuxize.com/post/how-to-install-pip-on-ubuntu-18.04/): You will need to install the latest version of python3.7.* . Why Python? because it's friggin awesome. You can use Python for developing desktop GUI applications, websites and web applications. Also, Python, as a high level programming language, allows you to focus on core functionality of the application by taking care of common programming tasks.
+
+
+#### How to run things on the terminal
+
+The terminal is an interface in which you can type and execute text based commands. It can be much faster to complete some tasks using a Terminal than with graphical applications and menus. Another benefit is allowing access to many more commands and scripts. A common terminal task of installing an application can be achieved within a single command, compared to navigating through the Software Centre or Synaptic Manager. Press *Ctrl + Alt + T* to open the terminal when using Ubuntu/Linux-mint
+
+##### Running Python
+
+There are 2 ways of running python that you should care about right now. The first way is pretty easy. Just open up a terminal end enter the command `python3`. This will open up a thing called a [REPL](https://codewith.mu/en/tutorials/1.0/repl). Basically it lets you just type in python commands and then it executes things. This is really useful if you just want to quickly calculate something or try out a piece of code.
+
+
+Of course if you've written some awesome code you would want to save it somewhere so you can run it whenever you want. For now let's assume you don't know your way around linux (if you do, that's cool. But for now we'll keep it simple). Try this out:
+
+- Open up a text editer and make a file that looks like this: `print("Welcome to Umuzi!")`
+- Save the file as `hellp.py` in your home directory (directory means folder). Now open up a terminal and type in `python3.7 hello.py`
+
+
+##### Running programs
+
+**Side Note**: update and upgrade your computer every chance you get, I do it every week. On your terminal run `sudo apt update` to update your machine, and if you need to upgrade the machine along with the apps, please run `sudo apt upgrade`. As a side note, the terminal is super powerful. You'll get quite familiar with it as time goes on. Another useful trick you might want right now is this one:
```

Comment:
Not sure that this side note is useful here. It should probably be with the linux specific stuff. It's better to let people focus during bootcamp.

Just show them what they need. Think KISS.